src/gui/cookiesdialog.h

```diff
@@ -1,6 +1,6 @@
 /*
- * Bittorrent Client using Qt4 and libtorrent.
- * Copyright (C) 2010  Christophe Dumez
```

Comment:
Rather, it's about the GitHub. When you look at the overall changes, it merges them, like this one commit. Let's look at the individual commits - it looks fine.